packages/smooth_app/lib/pages/scan/scan_page_helper.dart

```diff
@@ -20,5 +20,4 @@ Future<void> openPersonalizedRankingPage(BuildContext context) async {
       ),
     ),
   );
-  await model.refresh();
```

Comment:
Again, what's wrong with `await refresh()`?

Reply:
> Heyy @cli1005 I just tested it. The core works, the card doesn't disapper but when you finish the product the new card doesn't appear

#1973 persists... I have reopened the issue for adding newly added product into carousel

Reply:
the refresh will clear all barcodes first and re-add all product found, that's why we lost those "Create new" card